easycukes-scm/src/main/java/com/worldline/easycukes/scm/stepdefs/SCMStepdefs.java

```diff
@@ -62,7 +78,7 @@ public void cloneAGitRepository(String url, String target) throws Exception {
             username = configuration.getValues().credentials.getLogin() != null ? configuration.getValues().credentials.getLogin() : "";
             password = configuration.getValues().credentials.getPassword() != null ? configuration.getValues().credentials.getPassword() : "";
         }
-        final String fullPath = baseUrl + DataInjector.injectData(url);
+        final String fullPath = baseUrl.replace(SCM_BASE_URL, "") + DataInjector.injectData(url);        

```

Comment:
Actually there were sonar lint issue for this ;-
"Define a constant instead of duplicating this literal". To make this generic and avoid sonar issues Can we configure Error message in future ? 
For now will log messages directly.